### ISSUE TITLE
Fatal error in , line 0

### ISSUE THREAD

**Issue by Jamyz** [FlareSolverr/FlareSolverr#249]
### Environment

* **2.1**:
* **Operating system**:Raspbian GNU/Linux 10 (buster)
* **Are you using Docker**: yes
* **Are you using a proxy or VPN?** no
* **Are you using Captcha Solver:** no

### Description

I create a docker-compose.yml
inside i have :

---
version: "2.1"
services:
  flaresolverr:
    # DockerHub mirror flaresolverr/flaresolverr:latest
    image: ghcr.io/flaresolverr/flaresolverr:latest
    container_name: flaresolverr
    environment:
      - LOG_LEVEL=${LOG_LEVEL:-info}
      - LOG_HTML=${LOG_HTML:-false}
      - CAPTCHA_SOLVER=${CAPTCHA_SOLVER:-none}
      - TZ=Europe/London
    ports:
      - "${PORT:-8191}:8191"
    restart: unless-stopped 

When i launch docker-compose up -d he create correctly the container.
But when i try to go to 192.168.0.112:8191 nothing and logs are strange

### Logged Error Messages

```
#
# Fatal error in , line 0
# unreachable code
#
#
#
#FailureMessage Object: 0x7eae058c


#
# Fatal error in , line 0
# unreachable code
#
#
#
#FailureMessage Object: 0x7eefc58c


```

**Comment by ngosang** [comment]
Could you try this fix? https://github.com/FlareSolverr/FlareSolverr/issues/211#issuecomment-962504356

**Comment by Jamyz** [comment]
> Could you try this fix? [#211 (comment)](https://github.com/FlareSolverr/FlareSolverr/issues/211#issuecomment-962504356)

Great !
i use : 
`wget https://ftp.debian.org/debian/pool/main/libs/libseccomp/libseccomp2_2.5.1-1_armhf.deb
sudo dpkg -i libseccomp2_2.5.1-1_armhf.deb
rm libseccomp2_2.5.1-1_armhf.deb`
